src/fuzz/fuzz-journald-kmsg.c

```diff
@@ -0,0 +1,29 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include "fuzz.h"
+#include "journald-kmsg.h"
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+        Server s = {};
+        _cleanup_free_ char *buffer = NULL;
+
+        if (size == 0)
+                return 0;
+
+        s = (Server) {
+                .native_fd = -1,
+                .stdout_fd = -1,
+                .dev_kmsg_fd = -1,
+                .audit_fd = -1,
+                .hostname_fd = -1,
+                .notify_fd = -1,
+                .storage = STORAGE_NONE,
+        };
+        assert_se(sd_event_default(&s.event) >= 0);
+        buffer = memdup(data, size);
+        assert_se(buffer);
```

Comment:
Merge those two lines?

Reply:
As far as I know, Coverity doesn't like assignments mixed with assertions. d9fb7afb4890a93db was supposed to fix this but apparently cases like that haven't been covered yet.

Reply:
Ah, OK. NVM then.